e2e-tests/tests/upgrade-minor/01-create-cluster.yaml

```diff
@@ -13,25 +13,25 @@ commands:
       curl -s "https://raw.githubusercontent.com/percona/percona-postgresql-operator/v${lower_ver}/deploy/cr.yaml" >"${TEMP_DIR}/cr_lower.yaml"
 
       # get PG version which we want to test
-      pg_ver=$(echo "$IMAGE_POSTGRESQL" | cut -d':' -f2 | grep -oE '\-ppg[0-9]+-' | grep -oE '[0-9]+' | tr -d '-')
+      pg_ver=$PG_VER
```

Comment:
Just make sure that these versions are the same and that there was no intention that we provide image with the different version.
Can we just use PG_VER and not set another variable?

Reply:
> Can we just use PG_VER and not set another variable?
Well, we could, but later we have `pg_ver=$((pg_ver - 1))`

Reply:
> Just make sure that these versions are the same and that there was no intention that we provide image with the different version.
Well, we set PG_VER [here](https://github.com/Percona-Lab/jenkins-pipelines/blob/125fa091e5f8f0b4f6321edf1883f25e42825d7d/cloud/jenkins/pgo_operator_gke_latest.groovy#L230C90-L230C104) which is basically a more robust way than the one I'm obsoleting right now, because it works with minor numbers, as well.

Reply:
Hi, I would add passed param verification or modification. We rely that this is major version without minor (16 but not 16.3 or modify 16.3 to 16).